plugins/httpjson/httpjson.go

```diff
@@ -126,7 +133,7 @@ func (h *HttpJson) gatherServer(acc plugins.Accumulator, service Service, server
 		return err
 	}
 
-	var jsonOut interface{}
+	var jsonOut map[string]interface{}
```

Comment:
Hm, this plugin wasn't aware of array JSON. I think this should go to another PR (along with support for string values, do you know why the plugin supports only floats, btw?).